inst/tutorials/vaccines/vaccines.Rmd

```diff
@@ -0,0 +1,332 @@
+              answer("2000"),
+              answer("7000"),
+              answer("8000", correct = TRUE), correct = "Correct-- 8000 boosters are administered to the target population exactly 12 months after the third dose.", allow_retry = T)
+)
+
+```
+
+## Implement a mass vaccination strategy
+
+Use this space to implement a mass vaccination strategy that:
+
+- Is targeted at children under 5 years old
+- Includes a booster delivered 5 months after the first three doses
+- Reaches a coverage level of 50% for the first three doses and 75% for the booster dose.
+
+Plot incidence and the doses delivered for this model.
+
+```{r space_to_work, exercise = TRUE}
+
+
+
+```
+
+```{r mass_vax_example_questions, echo= FALSE}
+quiz(caption = "Mass Vaccination Example",
+          question("What modelled incidence does the vaccine model achieve 1.5 years into the simulation??",
+              answer("5 clinical cases per 1000"),
+              answer("2.5 clinical cases per 1000", correct = TRUE),
+              answer("4 clinical cases per 1000"),
+              answer("2.0 clinical cases per 1000"), correct = 'Correct-- notice how clinical incidence reaches a low during this period.', allow_retry = T),
+     question("About how many dose ~3s are administered in the vaccine simulation?",
+              answer("750"),
+              answer("1500"),
+              answer("1200", correct = TRUE),
+              answer("400"), allow_retry = T)
+     )
+
+```
+## Age-based vaccination
+
+Here, we will try an age-based strategy. With an age-based vaccine strategy, vaccines are administered to people at specific ages, rather than to the whole population at once.
+
+```{r age_based, exercise = TRUE}
+# Add RTS,S EPI strategy
+rtssepiparams <- set_pev_epi(
+  simparams,
+  profile = rtss_profile, # We will model implementation of the RTSS vaccine.
+  timesteps = 1 * year, # Vaccination will begin at 1 year into the simulation.
+  coverages = 1, # Vaccine coverage is 100%.
+  min_wait = 0, # There is no minimum wait since the last vaccination.
+  age = 5 * month, # Individuals will be vaccinated once they reach 5 months of age.
+  booster_timestep = 12 * month, # The booster is administered 12 months following the third dose. 
+  booster_coverage = 0.95, # 95% of those vaccinated with the primary series will be boosted.
+  booster_profile = list(rtss_booster_profile) # We will model implementation of the RTSS booster.
+)
+
+output <- run_simulation(timesteps = sim_length * 2, parameters = rtssepiparams)
+
+
+plot_inci(output)
+plot_prev(output)
+
+plot_doses(output)
+```
+
+Notice how, with an age-based strategy, vaccines are administered continuously throughout the simulation period, rather than at a few fixed points.
+
+
+```{r age_based_example_questions, echo= FALSE}
+quiz(caption = "Age-based vaccination",
+          question("About how many dose #1s are administered 50 months into the simulation?",
+              answer("55 doses", correct= TRUE),
```

Comment:
![image](https://github.com/mrc-ide/malsimtutorial/assets/73181123/8b37c1d6-2e02-4fb8-ac93-395885152ae5)

It looks to me more like 30 ish...? I guess there's variation in outputs, but could be worth adjusting the possible answers?

Reply:
Yes good point I changed the model setup which could have shifted the outputs-- anyway, these questions are deleted now for other reasons!